test/spec/modules/riseBidAdapter_spec.js

```diff
@@ -195,6 +195,12 @@ describe('riseAdapter', function () {
       expect(request.data.bids[1].mediaType).to.equal(BANNER)
     });
 
+    it('should send the correct currency in bid request', function () {
+      const expectedCurrency = 'USD';
+      const request = spec.buildRequests(bidRequests, bidderRequest)
+      expect(request.data.bids[0].currency).to.equal(expectedCurrency);
+    });
+
     it('should respect syncEnabled option', function() {
```

Comment:
add a test of EUR currency as a bidder param

Reply:
Done